cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperatorMockTest.java

```diff
@@ -230,45 +121,45 @@ public static void after() {
-                        .withReplicas(kafkaReplicas)
-                        .withStorage(kafkaStorage)
-                        .withListeners(new GenericKafkaListenerBuilder()
-                                    .withName("plain")
-                                    .withPort(9092)
-                                    .withType(KafkaListenerType.INTERNAL)
-                                    .withTls(false)
-                                    .build(),
-                                new GenericKafkaListenerBuilder()
-                                    .withName("tls")
-                                    .withPort(9093)
-                                    .withType(KafkaListenerType.INTERNAL)
-                                    .withTls(true)
-                                    .build())
-                        .withResources(resources)
-                    .endKafka()
-                    .withNewZookeeper()
-                        .withReplicas(zkReplicas)
-                        .withStorage(zkStorage)
-                    .endZookeeper()
-                    .withNewEntityOperator()
-                        .withNewTopicOperator()
-                        .endTopicOperator()
-                        .withNewUserOperator()
-                        .endUserOperator()
-                    .endEntityOperator()
-                .endSpec()
-                .build();
+                 .withNewMetadata()
+                     .withName(CLUSTER_NAME)
+                     .withNamespace(NAMESPACE)
+                 .endMetadata()
+                 .withNewSpec()
+                     .withNewKafka()
+                         .withConfig(new HashMap<>())
+                         .withReplicas(3)
+                         .withListeners(new GenericKafkaListenerBuilder()
+                             .withName("tls")
+                             .withPort(9092)
+                             .withType(KafkaListenerType.INTERNAL)
+                             .withTls(true)
+                             .build())
+                         .withNewPersistentClaimStorage()
+                         .withSize("123")
+                         .withStorageClass("foo")
+                         .withDeleteClaim(true)
+                         .endPersistentClaimStorage()
+                     .endKafka()
+                     .withNewZookeeper()
+                         .withReplicas(3)
+                        .withNewPersistentClaimStorage()
+                         .withSize("123")
+                         .withStorageClass("foo")
+                         .withDeleteClaim(true)
+                         .endPersistentClaimStorage()
+                         .endZookeeper()
+                 .endSpec()
+                 .build();
+
+        // Configure replicas and storage
+        this.kafkaReplicas = cluster.getSpec().getKafka().getReplicas();
+        this.kafkaStorage = cluster.getSpec().getKafka().getStorage();
```

Comment:
You know what the number of replicas and storage is so you can probably define it is `static final` constants?

Reply:
Resolved: I kept the storage parameter though since its being used throughout the code